cypress/support/fragments/data-export/dataExportResults.js

```diff
@@ -13,5 +13,24 @@ export default {
       }),
       MultiColumnListCell({ 'row': 0, 'column': defaultJobProfile }).exists(),
     ]);
-  }
+  },
+
+  verifySuccessExportResultCells() {
+    const dateString = /\d{1,2}\/\d{1,2}\/\d{4},\s\d{2}:\d{2}\s\w{2}/gm;
+    return cy.do([
+      MultiColumnListCell({ 'row': 0, columnIndex: 1 }).is({ content: 'Completed' }),
+      MultiColumnListCell({ 'row': 0, columnIndex: 2 }).perform(element => {
+        expect(element.innerText).to.match(/\d{1,4}/gm);
+      }),
+      MultiColumnListCell({ 'row': 0, columnIndex: 3 }).is({ content: '' }),
+      MultiColumnListCell({ 'row': 0, columnIndex: 4 }).is({ content: defaultJobProfile }),
+      MultiColumnListCell({ 'row': 0, columnIndex: 5 }).perform(element => {
```

Comment:
as I remember you have experience in implementation of steps like "Ended running: date and time of the completion of the export job"

Reply:
Do you think it is need this? I think match to regular expression will be ok

Reply:
such extra check can help to find issue with time resolving